input/postgres/relation_stats.go

```diff
@@ -65,10 +66,11 @@ SELECT s.indexrelid,
 			 COALESCE(sio.idx_blks_hit, 0)
 	FROM pg_catalog.pg_stat_user_indexes s
 			 LEFT JOIN pg_catalog.pg_statio_user_indexes sio USING (indexrelid)
-	WHERE s.indexrelid NOT IN (SELECT relid FROM locked_relids)
+ WHERE s.indexrelid NOT IN (SELECT relid FROM locked_relids)
+			 AND ($1 = '' OR (schemaname || '.' || relname) !~* $1)
```

Comment:
```suggestion
			 AND ($1 = '' OR (s.schemaname || '.' || s.relname) !~* $1)
```

Looks like this needs to be schema qualified, since both `pg_stat_user_indexes` and `pg_statio_user_indexes` have these columns